courses/free-courses-ar.md

```diff
@@ -19,6 +18,7 @@
 * [Game Development](#game-development)
 * [Git](#git)
 * [HTML and CSS](#html-and-css)
+    * [Bootstrap](#bootstrap)
```

Comment:
Why this change in TOC but without move the resources in listing?

We need mantain the structure consistently

Reply:
Ok 
Sry